tests/test_vector_comprehensive.py

```
@@ -0,0 +1,544 @@
+        # Value 3 should be masked
+        self.assertTrue(isinstance(v78, Vector))
+
+        # Test int() with shift parameter
+        v79 = Vector([0, 1, 2, 3])
+        v80 = v79.int(top=(3, 3, 3, 3), shift=True, remask=True)
+        self.assertTrue(isinstance(v80, Vector))
+
+        # Test as_index_and_mask with masked values
+        v81 = Vector([0, 1, 2])
+        v81 = v81.mask_where_component_le(0, 1)
+        idx3, mask3 = v81.as_index_and_mask()
```

Ruff (0.14.7)

Unpacked variable `mask3` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)